scripts/upgrade

```diff
@@ -43,6 +40,28 @@ then
     }
 fi
 
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
+
+#=================================================
+# STOP SYSTEMD SERVICE
+#=================================================
+ynh_script_progression --message="Stopping a systemd service..." --weight=1
+
+# We stop the service before to set ynh_clean_setup
+ynh_systemd_action --service_name=$app --action="stop"
+
```

Comment:
Well why do you stop the service after the backup. When you do this you won't be sure that you keep the data integrity when you run the back at the same time that the service is running.

Reply:
You should also express this concerns in the [example_yn](https://github.com/YunoHost/example_ynh/blob/master/scripts/upgrade#L38-L60) repo as it is the recommended order (first backup then stop service)